webhook_xblock/static/html/webhook_xblock_edit.html

```diff
@@ -0,0 +1,69 @@
+        <label class="label setting-label" for="frequency">Frequency</label>
+        <select class="input setting-input" style="width: 550px;" list="frequency-options" name="frequency" id="frequency" value="{frequency}" required>
+            <option value="one-time">One time</option>
+            <option value="always">Always</option>
+            <option value="sent-by-student">Let the student send the information</option>
+        </select>
+      </div>
+      <span class="tip setting-help">Indicate how often should we send the information</span>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="button_text">Button text</label>
+        <input class="input setting-input" style="width: 550px;" name="button_text" id="button_text" value="{button_text}" type="text" />
+      </div>
+      <span class="tip setting-help">Button inner text, in case the student is<br>allowed to decide when to send the payload.</span>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="component_text">Additional text</label>
+        <input class="input setting-input" name="component_text_value" id="component_text_value" value="{component_text}" type="hidden"/>
+        <textarea class="input setting-input" rows="1" style="width: 550px;" name="component_text" id="component_text" type="text"></textarea>
+      </div>
+      <span class="tip setting-help">Additional text to show in this component to the student<br>HTML formatting.</span>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="extra_info">Extra Information</label>
+        <input class="input setting-input" name="extra_info_value" id="extra_info_value" value="{extra_info}" type="hidden"/>
+        <textarea class="input setting-input" rows="1" name="extra_info" id="extra_info" style="width: 550px;" type="text"></textarea>
+      </div>
+      <span class="tip setting-help">Extra information to be sent in the payload.<br>Enter the fields in this format: &#123;"field_name": value, "another_field_name": another_value&#125;</span>
+    </li>
+    <li class="field comp-setting-entry is-set">
+        <div class="wrapper-comp-setting">
+          <label class="label setting-label" for="send_course_grade">Send Course Grade</label>
+          <input class="input setting-input" style="margin-left: -270pt;" name="send_course_grade" id="send_course_grade" value="{send_course_grade}" type="checkbox" />
+        </div>
+        <span class="tip setting-help">Send the student's course grade in the payload.</span>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="send_async">Send in Async Task</label>
```

Comment:
where can we see this? it's not in the screenshot

Reply:
@mariajgrimaldi you are right, let me add the rest of the screenshots